cvat/apps/dataset_manager/bindings.py

```diff
@@ -2151,10 +2151,64 @@ def reduce_fn(acc, v):
                 raise CvatImportError("Image {}: can't import annotation "
                     "#{} ({}): {}".format(item.id, idx, ann.type.name, e)) from e
 
-    for track in tracks.values():
-        track['elements'] = list(track['elements'].values())
-        instance_data.add_track(instance_data.Track(**track))
+    def append_necessary_outside_attribute(shapes):
```

Comment:
```suggestion
    def _validate_track_shapes(shapes):
```
I suggest that it is renamed to something more general or more close to the actual actions done.